src/Learning/KWData/KWMTDatabase.h

```diff
@@ -222,15 +226,24 @@ class KWMTDatabase : public KWDatabase
 	// et tables secondaires
 	mutable KWMTDatabaseMapping* mainMultiTableMapping;
 
-	// Mapping racine des tables secondaires (reference a un sous-ensemble des mappings du tableau
-	// oaMultiTableMappings)
-	mutable ObjectArray oaRootRefTableMappings;
+	// Mapping racine des tables externes (sous-ensemble des mappings du tableau oaMultiTableMappings)
+	mutable ObjectArray oaRootReferenceTableMappings;
 
 	// Mapping multi-tables: tableau exhaustif de tous les mappings (permet une interface d'edition "a plat")
 	// Attention, le mapping de la classe principale est toujours integree comme premier element du tableau,
 	// et ne doit toujours etre synchronise
 	mutable ObjectArray oaMultiTableMappings;
 
+	// Warnings pour le cas des tables externes non utilises, gardes pour generer des warnings lors du Check
```

Comment:
"non utilise**e**s" (feminine, for "des tables").

Reply:
Corrigé